src/states/elder/mod.rs

```diff
@@ -1559,6 +1553,14 @@ impl Base for Elder {
                 );
                 self.direct_msg_backlog.push((p2p_node, msg));
             }
+            msg => {
+                trace!(
```

Comment:
The same decision was not made for `dispatch_routing_message` because match was not exhausitve, was it right?
(I guess it does not matter very much for this intermediate state because it cannot happen).